hibernate-envers/src/test/resources/mappings/embeddedComponents/embedded.hbm.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE hibernate-configuration PUBLIC
+        "-//Hibernate/Hibernate Configuration DTD 3.0//EN"
+        "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+<hibernate-configuration>
+	<session-factory>
+		<!-- Database connection settings -->
+		<property name="dialect">org.hibernate.dialect.H2Dialect</property>
+		<property name="connection.url">jdbc:h2:mem:envers</property>
+		<property name="connection.driver_class">org.h2.Driver</property>
+		<property name="connection.username">sa</property>
+		<property name="connection.password"></property>
+		<property name="hibernate.hbm2ddl.auto">update</property>
+		<property name="show_sql">true</property>
+
+		<mapping class="org.hibernate.envers.test.embeddedid.OwnerOfRelationCode" />
+		<mapping class="org.hibernate.envers.test.embeddedid.CompositeEntity" />
+
+	</session-factory>
+</hibernate-configuration>
```

Comment:
As mentioned earlier, this is completely unnecessary, so you should be able to safely remove this entirely.  You expose the classes the test should load by overriding the getter methods and the other hibernate property settings are handled automatically by the test suite.